src/components/kitchenChefSection/KitchenChef.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { useQuery } from "styles/breakpoints";
+import { StaticImage } from "gatsby-plugin-image";
+import { Box, GridWrapper, SectionWrapper } from "components";
+import SectionHeader from "components/sectionHeader/SectionHeader";
+import BlackButton from "components/buttons/BlackButton";
+import { FlexWrapper } from "components/wrappers/FlexWrapper";
+
+export const KitchenChef: React.FC = () => {
+  const { isMobile, isTablet } = useQuery();
+  return (
+    <SectionWrapper>
+      <GridWrapper gridTemplateColumns={isTablet ? "1fr" : "1fr 1fr"}>
+        <Box>
+          <SectionHeader
+            header={"Everyone can be a chef in their own kitchen"}
+            paragraph={
+              "Lorem ipsum dolor sit amet, consectetuipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqut enim ad minim "
+            }
+          />
+          <FlexWrapper
+            ml={isMobile ? "s0" : "s70"}
+            justifyContent={isMobile ? "center" : undefined}
```

Comment:
bad practise to pass an undefined. check the default value and pass it insted of undefined